src/test/scala/edu/ie3/osmogrid/utils/SolverSpec.scala

```diff
@@ -164,7 +164,7 @@ class SolverSpec extends UnitSpec with MvTestData {
                 usedConnections shouldBe List(connectionA, connectionB).flatten
                 removedEdge shouldBe expectedRemovedEdgeA
                 addedWeight.getValue.doubleValue() shouldBe weightA
-              case _ => throw new Error("This should not happen!")
+              case null => throw new Error("This should not happen!")
```

Comment:
In my understanding of [this](https://docs.scala-lang.org/tour/pattern-matching.html) pattern matching still supports the catch all case, right? So whats the reason to swith to `null`?

Reply:
This was a suggestion from the compiler. Since this code is only reached, if we have a `null` value.